.github/workflows/mega-linter.yml

```diff
@@ -60,5 +81,35 @@ jobs:
         with:
           name: MegaLinter reports
           path: |
-            report
-            mega-linter.log
+            megalinter-reports
+
+      # Create pull request if applicable
+      # Note: This step will fail if megalinter tries to modify any file in the .github/workflow
+      # directory. Github has a protection that workflows cant modify workflows. Please fix those
```

Comment:
```suggestion
      # directory. Github has a protection that workflows can't modify workflows. Please fix those
```

Reply:
addressed in e1250bd6ccf32489638864a367c380f8b73589cc